### ISSUE TITLE
Sparse accessor is causing filament.js to crash

### ISSUE THREAD

**Issue by elalish** [google/filament#1727]
**Describe the bug**
Loading a model with a sparse accessor in filament.js fails: 
```
filament-viewer.js:145 Uncaught (in promise) TypeError: Cannot read property 'loadResources' of null
    at finalize (filament-viewer.js:145)
    at new Promise (<anonymous>)
    at HTMLElement.[scenario] (filament-viewer.js:143)
```

**To Reproduce**
This is coming from the model-viewer fidelity tests, and the problem model is khronos-SimpleSparseAccessor. Reproduces on Chrome, Safari, Firefox on both OSX and Linux.

**Comment by prideout** [comment]
On native, we at least print "Sparse accessors not yet supported" to the console before bailing out.  :) This is a known limitation but maybe it can be implemented without too much work -- let me think about it.

**Comment by prideout** [comment]
I created a cgltf ticket for this:
https://github.com/jkuhlmann/cgltf/issues/83

**Comment by prideout** [comment]
Should be fixed with #1769 1758
